.gitmodules

```diff
@@ -16,8 +16,8 @@
   branch = develop
 [submodule "CMEPS"]
   path = CMEPS-interface/CMEPS
-  url = https://github.com/NOAA-EMC/CMEPS
-  branch = emc/develop
+  url = https://github.com/uturuncoglu/CMEPS
```

Comment:
CMEPS needs to point back to noaa-emc repo too

Reply:
@junwang-noaa this is updated now.